### PR TITLE
Fix - TRSV2-522 - Pingdom causing hanging websites

### PR DESCRIPTION
**Pull request by chris-pettinga**

### CODE REVIEW

File: trade_remedies_api/core/healthcheck.py [hunk 1]
```diff
@@ -81,14 +88,13 @@ def application_service_health():
...
-                response_times.append(response_time)
-            except Exception as err:
-                return _pingdom_custom_status_html_wrapper(f"Error: {str(err)}", 0)
+    for service_check in services:
+        try:
+            _, response_time = service_check()
+            response_times.append(response_time)
+        except Exception as err:
+            sentry_sdk.capture_exception(err)
+            return _pingdom_custom_status_html_wrapper(f"Error: {str(err)}", 0)
```
Comment (osimuka) [inline trade_remedies_api/core/healthcheck.py:97]: @chris-pettinga is there a reason to change from the thread pool?

Comment (osimuka) [reply trade_remedies_api/core/healthcheck.py:97]: @chris-pettinga I think the `test cases` can be updated to match the new checks

Comment (chris-pettinga) [reply trade_remedies_api/core/healthcheck.py:97]: @osimuka I think the threadpool was causing timeouts causing UAT to go down

Comment (osimuka) [reply trade_remedies_api/core/healthcheck.py:97]: oh ok, no worries

**Review by osimuka — APPROVED** [review]

**Pull request merged**